src/Simplex/Chat/Help.hs

```diff
@@ -11,27 +11,26 @@ chatHelpInfo :: [StyledString]
 chatHelpInfo =
   map
     styleMarkdown
-    [ Markdown (Colored Cyan) "Using Simplex chat prototype.",
+    [ highlight "Using Simplex chat prototype.",
       "Follow these steps to set up a connection:",
       "",
-      Markdown (Colored Green) "Step 1: " <> Markdown (Colored Cyan) "/add bob" <> " -- Alice adds her contact, Bob (she can use any name).",
+      Markdown (Colored Green) "Step 1: " <> highlight "/add" <> " -- Alice adds a contact.",
       indent <> "Alice should send the invitation printed by the /add command",
       indent <> "to her contact, Bob, out-of-band, via any trusted channel.",
       "",
-      Markdown (Colored Green) "Step 2: " <> Markdown (Colored Cyan) "/connect alice <invitation>" <> " -- Bob accepts the invitation.",
-      indent <> "Bob also can use any name for his contact, Alice,",
-      indent <> "followed by the invitation he received out-of-band.",
+      Markdown (Colored Green) "Step 2: " <> highlight "/connect <invitation>" <> " -- Bob accepts the invitation.",
```

Comment:
re-reading these steps and it's kind of an awkward transition from reader - "Follow these steps" - to Alice and Bob. Not significant though, maybe add a TODO for another pr